QUEST_LV_0200.tsv

```diff
@@ -6209,15 +6209,15 @@ QUEST_LV_0200_20150406_006212	Not being affirmative is a big weakness. {nl}I thi
 QUEST_LV_0200_20150406_006213	He's a greedy old man so there must be something. I'm sure about it.
 QUEST_LV_0200_20150406_006214	{memo X}Place the poison pot, lure the monster to it, then kill it. {nl}Bring it to me when enough is collected.
 QUEST_LV_0200_20150406_006215	I'm sorry for asking something like this. {nl}But please feel pity for us and help us out a little more.
-QUEST_LV_0200_20150406_006216	I'm thinking about borrowing farm tools from farmers nearby for the lacking weapons. {nl}You are a famour revelator so it will be fairly easy for you to borrow the tools.
+QUEST_LV_0200_20150406_006216	$I'm thinking about borrowing farm tools from the farmers nearby for improvised weapons. {nl}You are a famous Revelator, so it should be fairly easy for you to convince them.
 QUEST_LV_0200_20150406_006217	If you pursuade them with the letter I give you, they will understand.
-QUEST_LV_0200_20150406_006218	This much.. Thank you very much. {nl}Oh, if you have time, I hope you can help my collagues as well.
+QUEST_LV_0200_20150406_006218	$This many.. Thank you so much. {nl}Oh, if you have time, I hope you can help my colleagues as well.
 QUEST_LV_0200_20150406_006219	I am Horacius' colleague. I am in charge of the food supplies. {nl}The promised day is coming but we don't have enough manpower. Can you help us?
 QUEST_LV_0200_20150406_006220	We will make it with Horacius. {nl}Everyone is exhauted from the unreasonable farm rent fees.
 QUEST_LV_0200_20150406_006221	Now that the Baron secretary who used to protect us is gone.. {nl}We only have our teo hands to trust.
 QUEST_LV_0200_20150406_006222	Then please get the Nioles with roots from Riaum knoll. {nl}We can use it with potatos to make soup and it is also used as medicine.
-QUEST_LV_0200_20150406_006223	Even if the baron did something right, it's all for him to live. {nl}No one can point their finger at us.
-QUEST_LV_0200_20150406_006224	And.. dried foods are better to keep for a long period. {nl}The jerkeys especially keeps you full even with little intake so I'm trying to gather as much as I can.
+QUEST_LV_0200_20150406_006223	$Even if the baron did something right, he only did it for himself. {nl}No one can blame us for what we're doing.
+QUEST_LV_0200_20150406_006224	$Dried food is better to keep it for a long time.{nl}Especially, you will feel full when you consume only a little amount of beef jerky.{nl}That's why we are accumulating them a lot.
```

Comment:
Dried food has a longer shelf life.{nl}You'll feel full even when you consume only a little amount of beef jerky.{nl}That's why we are accumulating a lot of them.